src/main/java/net/coderbot/iris/mixin/MixinGameRenderer.java

```diff
@@ -24,4 +34,17 @@
 		Iris.logger.info("GPU: " + GlUtil.getRenderer() + " (Supports OpenGL " + GlUtil.getOpenGLVersion() + ")");
 		Iris.logger.info("OS: " + System.getProperty("os.name"));
 	}
+
+	@Inject(method = "getProjectionMatrix", at = @At(value = "INVOKE", target = "Lcom/mojang/math/Matrix4f;multiply(Lcom/mojang/math/Matrix4f;)V"), locals = LocalCapture.CAPTURE_FAILHARD)
+    private void scaleHandDepth(Camera camera, float f, boolean bl, CallbackInfoReturnable<Matrix4f> cir, PoseStack poseStack) {
+        if (HandRenderer.isActive()) {
+            // This value is taken directly from Shaders Mod.
+            poseStack.scale(1F, 1F, 0.125F);
```

Comment:
Would it be possible to move this to HandRenderer as ie. a pre-multiply on the matrix we get from getProjectionMatrix? Combined with other changes, we could remove the HandRenderer.ACTIVE global state.

Reply:
I've moved the whole projection matrix code to the hand renderer.